### DISCUSSION TITLE
Any Particle System?

### DISCUSSION THREAD

**Discussion by jozadaquebatista** [o3de/o3de#17463] [Q&A] [answered]
Since O3DE originates from Amazon Lumberyard, is there any plan to make it available or is there already some type of particle system made available by the engine without necessarily using third-party software?

**Comment by AMZN-daimini** [comment, marked as answer]
Hello! O3DE does not have a native particle system as of right now. PopcornFX has an O3DE Gem that is what most projects I see use, but it probably falls under the third-party software part of your question.

Lumberyard's particle system was built off of a different renderer so I doubt it would be simple to port over. It would probably be more worthwhile to just build a new system from scratch at that point.

**Reply by HydrogenC** [reply]
Is this prioritized yet? A native particle system is a fundamental functionality for most engines.

**Reply by someStranger1** [reply]
I'l give some statistics and sure after that - question can be closed:

git shortlog -s --since=2022-11-01 --until=2022-11-30
34 commiters
81 contributors

git shortlog -s --since=2023-11-01 --until=2023-11-30
13 commiters
26 contributors

Conclusion: the number of developers has decreased by ~3 times

P.S.
After raising same question, my git account and another user's account who asked same question earlier - were blocked. So - be carefull!
